Packs/Cymulate/Integrations/Cymulate_v2/README.md

```diff
@@ -0,0 +1,1778 @@
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Starting immediate-threats assessment:
+>|id|success|
+>|---|---|
+>| id_678 | true |
+
+
+### cymulate-immediate-threat-stop
+***
+Stop a running immediate threats assessment.
+
+
+#### Base Command
+
+`cymulate-immediate-threat-stop`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Cymulate.ImmediateThreats.data | String | Immediate threats assessment stopping data. | 
+
+
+#### Command Example
+```!cymulate-immediate-threat-stop```
+
+#### Context Example
+```json
+{
+    "Cymulate": {
+        "ImmediateThreats": {
+            "Assessment": {
+                "data": "ok",
+                "success": true
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Stop immediate-threats assessment:
+>|data|success|
+>|---|---|
+>| ok | true |
+
+
+### cymulate-immediate-threat-status
+***
+Get immediate threats assessment status.
+
+
+#### Base Command
+
+`cymulate-immediate-threat-status`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| assessment_id | Assessment ID. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Cymulate.ImmediateThreats.id | String | Web application firewall assessment ID. | 
+
+
+#### Command Example
+```!cymulate-immediate-threat-status assessment_id="id_789"```
+
+#### Context Example
+```json
+{
+    "Cymulate": {
+        "ImmediateThreats": {
+            "Assessment": {
+                "addresses": null,
+                "categories": [
+                    "antivirus"
+                ],
+                "id": "id_789",
+                "inProgress": false,
+                "progress": 90
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Immediate-threats assessment status:
+>|categories|id|inProgress|progress|
+>|---|---|---|---|
+>| antivirus | id_789 | false | 90 |
+
+
+### cymulate-phishing-awareness-contacts-group-list
+***
+Get a list of contact groups.
+
+
+#### Base Command
+
+`cymulate-phishing-awareness-contacts-group-list`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Cymulate.Phishing.Groups.id | String | The ID of the phishing content group. | 
+| Cymulate.Phishing.Groups.name | String | Name of the phishing content group. | 
+| Cymulate.Phishing.Groups.client | String | The client of the phishing content group. | 
+| Cymulate.Phishing.Groups.canDelete | Boolean | Whether this group can be deleted. | 
+
+
+#### Command Example
+```!cymulate-phishing-awareness-contacts-group-list```
+
+#### Context Example
+```json
+{
+    "Cymulate": {
+        "Phishing": {
+            "Groups": [
+                {
+                    "__v": 0,
```

Comment:
It’s a random bug in Cymulate API, and it will be fixed soon by them. 
This field is not included in the context output and the reason it is visible in the `README` and the table is that this endpoint sends the `output` data from the API "as is" and does not parse any fields. 
Do you think it's better to handle this field differently?